docs/docs/faq/question_02.md

```diff
@@ -4,9 +4,7 @@ Yes. By default, whenever a simulation returns at least one `NaN` or `inf`, it i
 completely excluded from the training data. In other words, the simulation is simply 
 discarded.
 
-In cases where a very large fraction of simulations return `NaN` or `inf`, discarding 
-many simulations can be wasteful. Hence, it can be 
-beneficial to manually substitute the 'invalid' values with a reasonable replacement. 
+In cases where a very large fraction of simulations return `NaN` or `inf`, discarding many simulations can be wasteful. There are two options to deal with this: Either, you use the `RestrictionEstimator` to learn regions in parameter space that do not produce `NaN` or `inf`, see [here](https://www.mackelab.org/sbi/tutorial/08_restriction_estimator/). Alternatively, you can manually substitute the 'invalid' values with a reasonable replacement. 
 I.e., at the end of your simulation code, you search for invalid entries and replace 
 them with a floating point number. Importantly, in order for neural network training 
 work well, the floating point number should still be in a reasonable range, i.e. maybe a
```

Comment:
Should we mention the calibration kernels here? We have the code for in `snpe_base` but we have never tested this properly, have we? 
Or is this irrelevant because SNPE-B is not implemented?

Reply:
hm, not sure. It's indeed not tested well, so maybe let's not mention it yet? It's also not directly related to `invalid` data. (but calibration kernels can be used for snpe-c also)

Reply:
Yes, maybe not mention it here. But weren't calibration kernels introduced in SNPE-B exactly for that, for handling invalid data?

Reply:
yes, but in our code base they are not. They are used only to reweigh the loss, but `invalid` data is simply ignored. I think it would be confusing to people if we started talking about the `calibration_kernel` argument here.

Reply:
OK, agreed, then I think it is fine to just leave it as it is.